_episodes/01-numpy.md

```diff
@@ -1135,3 +1135,73 @@ the graphs will actually be squeezed together more closely.)
 > > {: .output}
 > {: .solution}
 {: .challenge}
+
+>## Change In Inflamation
+>
+>This patient data is _longitudinal_ in the sense that each row represents a
+>series of observations relating to one individual. This means that change
+>inflamation is a meaningful concept.
+>
+>The `numpy.diff()` function takes a NumPy array and returns the 
+>difference along a specified axis.
+>
+>Which axis would it make sense to use this function along?
```

Comment:
Can you add a blank blockquote line here to separate the question from the solution? I'm not sure if it's necessary for the markdown rendering, but for those reading the markdown it's helpful at least.